test/lib/model/test_classycat.py

```diff
@@ -0,0 +1,708 @@
+                        "labels": [
+                            "Politics",
+                            "Democracy & Election Integrity"
+                        ]
+                    },
+                    {
+                        "text": "बेहद गंभीर मामला है। खबरों के मुताबिक, दवा बनाने वाली बहुत सी कंपनियां जिनकी दवाएं टेस्ट में फेल हुईं, इन कंपनियों ने इलेक्टोरल बॉन्ड से चंदा दिया। जैसे \uD83D\uDC47 1. गुजरात की कंपनी 'टोरेंट फार्मास्यूटिकल' की बनाई 3 दवाएं ड्रग टेस्ट में फेल हुईं। इस कंपनी ने BJP को 61 करोड़ का चंदा दिया। 2. सिप्ला लिमिटेड कंपनी की बनाई दवाओं के 7 बार ड्रग टेस्ट फ़ेल हुए। इस कंपनी ने BJP को 37 करोड़ का चंदा दिया। 3. सन फ़ार्मा कंपनी की बनाई गई दवाओं के 6 बार ड्रग टेस्ट फ़ेल हुए। सन फार्मा ने BJP को 31.5 करोड़ रुपए का चंदा दिया। 4. गुजरात की कंपनी इंटास फ़ार्मास्युटिकल्स की बनाई गई दवा का भी ड्रग टेस्ट फ़ेल हुआ। इंटास फ़ार्मा ने 20 करोड़ के इलेक्टोरल बॉन्ड ख़रीदे और सारा चंदा BJP को दिया। 5. ग्लेनमार्क फ़ार्मा कंपनी की दवाओं के 6 ड्रग टेस्ट फ़ेल हुए। ग्लेनमार्क फ़ार्मा 9.75 करोड़ रुपए के बॉन्ड ख़रीदे और सारा चंदा BJP को दिया।",
+                        "labels": [
+                            "Corruption",
+                            "Health & Pharma",
+                            "Politics"
+                        ]
+                    },
+                    {
+                        "text": "യുഎഇ അബുദാബിയിലെയും ദുബായിലെയും എല്ലാ അറബ് മന്ത്രിമാരും അറബ് ഷേക്'മാരും റമദാൻ മാസത്തിൽ ഇന്ന് അബുദാബി ഹിന്ദു ക്ഷേത്രത്തിലെത്തി പ്രസാദത്തോടെ ക്ഷേത്രത്തിൽ റമദാൻ വ്രതം പൂർത്തിയാക്കി. (ഇതൊരു വലിയ അത്ഭുതമാണ്.) ഈ വീഡിയോ കാണുക.\uD83D\uDEA9\uD83D\uDEA9\uD83D\uDEA9\n\n",
+                        "labels": [
+                            "Foreign Policy",
+                            "Politics",
+                            "Communalism"
+                        ]
+                    }
+                ],
+                "languages": [
+                    "English",
+                    "Hindi",
+                    "Telugu",
+                    "Malayalam"
+                ]
+            }
+        )
+        openrouter_classify_mock.return_value = "<OUTPUT>\n<CATEGORIES_0>Politics;Communalism</CATEGORIES_0>\n<CATEGORIES_1>Politics;Communalism</CATEGORIES_1>\n</OUTPUT>"
+        classify_input = {
+            "model_name": "classycat__Model",
+            "body": {
+                "id": 1200,
+                "parameters": {
+                    "event_type": "classify",
+                    "schema_id": "4a026b82-4a16-440d-aed7-bec07af12205",
+                    "items": [
+                        {
+                            "id": "11",
+                            "text": "modi and bjp want to rule india by dividing people against each other"
+                        }
+                    ]
+                },
+                "callback_url": "http://host.docker.internal:9888"
```

Comment:
It looks like tests are failing at an earlier stage right now around getting `CLASSYCAT_BATCH_SIZE_LIMIT`, but I suspect the tests will also fail with a timeout from `http://host.docker.internal:9888` on Linux. This has been my experience. 

> If you are using [Docker-for-mac](https://docs.docker.com/docker-for-mac/networking/#there-is-no-docker0-bridge-on-macos#i-want-to-connect-from-a-container-to-a-service-on-the-host) or [Docker-for-Windows](https://docs.docker.com/docker-for-windows/networking/#there-is-no-docker0-bridge-on-windows#i-want-to-connect-from-a-container-to-a-service-on-the-host) 18.03+, connect to your mysql service using the host host.docker.internal (instead of the 127.0.0.1 in your connection string).
>
> If you are using Docker-for-Linux 20.10.0+, you can also use the host host.docker.internal if you started your Docker container with the --add-host host.docker.internal:host-gateway option, or added the following snippet in your docker-compose.yml file :
```
extra_hosts:
    - "host.docker.internal:host-gateway"
```
More at  https://stackoverflow.com/a/24326540

Reply:
thanks for flagging Scott. 

I think the tests failing bc of `CLASSYCAT_BATCH_SIZE_LIMIT` (or other en vars) is due to the fact that these new environment variables are not being correctly set in the test cases. I will look into what the issue is and try to figure it out.

re: callbacks not working -- I think that makes sense and is expected. the address mentioned there is a local server I have spun up to catch the callbacks from Presto. I did not commit the server with the PR code (but happy to do so if helpful), and therefore it doesn't exist to receive the results. 

I'm now realizing that the other tests have been mocking the callback functionality, and I have not done so for classycat. I will do that soon and update the PR. it should fix the timeout error.

Reply:
Yes, my personal preference as well is mocking callback functionality for intra-container tests